docs/README.md

```diff
@@ -11,51 +61,64 @@ These instructions will get you a copy of the project up and running on your loc
 What things you need to install the software and how to install them
 
 ```
-live-server
+@babel/core
+@babel/plugin-syntax-import-assertions
+babel-loader
+concurrently
+css-loader
+mini-css-extract-plugin
+node-sass
+sass-loader
+serve
+style-loader
+webpack
+webpack-cli
 ```
 
 ### Installing
 
-A step by step series of examples that tell you how to get a development env running
+Our application is using npm for package management. To install all the necessary package follow these steps:
 
-Install live-server globally if you haven't already
+1. Install dependencies
 
 ```
-npm install live-server -g
+npm install
 ```
 
-Then launch the server
+2. Then launch the server
 
 ```
-live-server
+npm run dev
 ```
 
-The page will render in a new window.
-
-## Deployment
+3. The page will render in a new window.
 
-This site is just a static website with html, css, and js so it should not need to be deployed past hosting somewhere
+### Deployment
 
-## Built With
+To deploy the package, enter the command:
 
-* [HTML5](https://developer.mozilla.org/en-US/docs/Glossary/HTML5) - The web framework used
-* [CSS](https://developer.mozilla.org/en-US/docs/Glossary/css) - The styling language used
-* [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript) - The programming language used
+```
+npm run build
+```
 
 ## Authors
 
-* **Chingu Voyages** - *Initial work* - [chingu-voyages](https://github.com/chingu-voyages)
-* **Emilie Echevin** -- *Developer* - [Emimint](https://github.com/Emimint)
-* **Emilio Rivera** -- *Developer* - [emilio12345](https://github.com/emilio12345) 
-* **Gabriela de Paula** -- *Developer* - [PaulaR-05](https://github.com/PaulaR-05) 
-* **Jim Medlock** - *Initial work* - [jmedlock](https://github.com/jdmedlock)
-* **Mikey Nichols** -- *Developer* - [mnichols08](https://github.com/mnichols08)
-* **Nazile Tag** -- *Developer* - [Nazile-Tag](https://github.com/Nazile-Tag) 
+- **Chingu Voyages** - _Initial work_ - [chingu-voyages](https://github.com/chingu-voyages)
+- **Emilie Echevin** -- _Developer_ - [Emimint](https://github.com/Emimint)
+- **Emilio Rivera** -- _Developer_ - [emilio12345](https://github.com/emilio12345)
+- **Gabriela de Paula** -- _Developer_ - [PaulaR-05](https://github.com/PaulaR-05)
+- **Jim Medlock** - _Initial work_ - [jmedlock](https://github.com/jdmedlock)
+- **Mikey Nichols** -- _Developer_ - [mnichols08](https://github.com/mnichols08)
+- **Nazile Tag** -- _Developer_ - [Nazile-Tag](https://github.com/Nazile-Tag)
 
-See also the list of [contributors](/CONTRIBUTORS.md) who participated in this project. 
+See also the list of [contributors](/CONTRIBUTORS.md) who participated in this project.
```

Comment:
I think we may have to change `/CONTRIBUTORS.md` to `/docs/CONTRIBUTORS.md` to fix the link.

I think we should still merge this code but create an bug report to fix this in another PR